packages/shared/src/components/modals/PushNotificationModal.tsx

```diff
@@ -1,10 +1,43 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { cloudinary } from '../../lib/image';
 import { Button, ButtonVariant } from '../buttons/Button';
 import { Justify } from '../utilities';
 import { Modal, ModalProps } from './common/Modal';
 import { NotificationPromptSource } from '../../lib/analytics';
 import { usePushNotificationMutation } from '../../hooks/notifications';
+import { useModalContext } from './common/types';
+
+const DrawerImg = () => {
+  const { isDrawer } = useModalContext();
+
+  if (!isDrawer) {
+    return null;
+  }
+
+  return (
+    <img
+      src={cloudinary.notifications.big}
+      alt="A sample browser notification"
+    />
+  );
+};
+
+const ModalImg = () => {
+  const { isDrawer } = useModalContext();
+
+  if (isDrawer) {
+    return null;
+  }
+
+  return (
+    <img
+      className="mx-auto my-14"
+      src={cloudinary.notifications.big}
+      alt="A sample browser notification"
+    />
+  );
+};
```

Comment:
Wait this is exactly the same, but one has classes?
Can't we just uniform them? *(or is it a placement issue?)

Reply:
I did before, but we render two elements at the same time. Then Ante suggested this approach to avoid the double elements. https://github.com/dailydotdev/apps/pull/2815#discussion_r1528325281